crates/matrix-sdk-ui/src/timeline/tests/read_receipts.rs

```diff
@@ -485,18 +486,17 @@ async fn test_read_receipts_updates_on_message_decryption() {
         )
         .await;
 
-    assert_eq!(timeline.controller.items().await.len(), 2);
```

Comment:
Why was this removed?

Reply:
Because it is not immediately true: we need to wait for the async events to happen. Given that in the rest of this test we check that the timeline stream is as we expect, I didn't think there was much benefit in this check. If we want to keep it, we can move it later so we know the async work has been processed.

Reply:
Ah ok, fine by me to remove it then.